code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -139,16 +139,26 @@
 													//WHO THE FUCK THOUGHT THAT WAS A GOOD FUCKING IDEA!?!?
 
 		if("howl", "howls")
-			var/M = handle_emote_param(param) //Check to see if the param is valid (mob with the param name is in view).
-			message = "<B>[src]</B> howls[M ? " at [M]" : ""]!"
-			playsound(loc, 'sound/goonstation/voice/howl.ogg', 100, 1, 10, frequency = get_age_pitch())
-			m_type = 2
+			if(isvulpkanin(src)) 				  //Need this again for some reason, although it's already checked earlier
```

Comment:
remove the pointless comments

Reply:
Also the check should not be needed due to line 59-62.

Reply:
I'd assumed it was good practice to comment stuff that may have a confusing reason to exist, and the emote code is pretty well commented, so I figured I'd explain it for future coders, but I can remove it

Reply:
Commenting on it's own is fine if it adds to the readability of the code. Placing a comment saying you don't know why this code exists is usually not a good comment since it will most likely only confuse readers more.

I use this rule for myself: If you need to comment code to make it readable then you're probably doing something wrong. However adding comments to complex code that you can't simplify is fine. Or to explain a proc and explain what it does etc.